tests/commands/docker/test_docker_cleanups.py

```diff
@@ -11,6 +11,7 @@
 
 @pytest.mark.docker
 @pytest.mark.slow
+@pytest.mark.makelivecalls
```

Comment:
this is not related to "Fix report"

Reply:
Understood, however this test should have been marked as for integration testing in 1.3.1, and was failing during my local testing of Issue #175 so I added the small change.